routing 翻译第一版

kzh4435 opened the pull request:
routing 翻译第一版

kzh4435 committed:
routing 翻译第一版

routing 翻译第一版

overtrue commented on routing.md:
```diff
@@ -1,39 +1,39 @@
+- [路由模型绑定](#route-model-binding)
+    - [隐式绑定](#implicit-binding)
+    - [显式绑定](#explicit-binding)
+- [跨站请求伪造](#form-method-spoofing)
```
这里不对，它只是模拟请求 method 而已，与跨站没有关系，也就是说相当于使用 get 来模拟 delete.

kzh4435 replied:
关于命名的问题，我是参考5.2的文档来的

summerblue replied:
对对，这是一个错误，超哥强悍

kzh4435 replied:
第一次做翻译文档，没有仔细检查。以后会改进的。

overtrue replied:
@kzh4435 加油！ ❤️

overtrue commented on routing.md:
```diff
@@ -1,39 +1,39 @@
+    - [必选路由参数](#required-parameters)
+    - [可选路由参数](#parameters-optional-parameters)
+- [命名路由](#named-routes)
+- [路由群组](#route-groups)
```
叫 `路由组` 也许会合适些

summerblue replied:
我觉得行，之前有点直译

kzh4435 replied:
好的，那就这样改

overtrue commented on routing.md:
```diff
@@ -1,39 +1,39 @@
+## 基本路由
 
-The most basic Laravel routes simply accept a URI and a `Closure`, providing a very simple and expressive method of defining routes:
+最基本的路由仅仅接受一个 URI 和一个 `闭包`，这里提供了一个非常简单优雅的定义路由的方法：
```
_"仅仅接受一个 URI 和一个 `闭包`"_  我觉得翻译成  _"只需要一个 `URI` 与一个  #`闭包`"_， 因为 `simply` 是**简单地**，所以作者想表达用起来简单，学习成本低。:smile: 个人观点，仅供参考。

summerblue replied:
@kzh4435 这个完全可以接受超哥的意见

kzh4435 replied:
是的，超哥的理解很透彻。我正在修改，估计要晚上才能修改完。

overtrue commented on routing.md:
```diff
@@ -1,39 +1,39 @@
+#### 默认路由文件
 
-All Laravel routes are defined in your route files, which are located in the `routes` directory. These files are automatically loaded by the framework. The `routes/web.php` file defines routes that are for your web interface. These routes are assigned the `web` middleware group, which provides features like session state and CSRF protection. The routes in `routes/api.php` are stateless and are as ...
+所有的Laravel路由定义都在 `routes` 目录。路由文件是框架自动加载的。 `routes/web.php` 文件定义了所有的web界面的路由。这些路由分配 给`web` 中间件，而这个中间件为 `Session` 和 `CSRF` 提供了保护功能。 `routes/api.php` 中的路由是没有状态的，它分配给了 `api` 中间件。
```
注意留白哦：

``` diff
- 所有的Laravel路由
+ 所有的 Laravel 路由
```

``` diff
- 所有的web界面
+ 所有的 web 界面
```

kzh4435 replied:
谢谢提醒。

nauxliu replied:
``` diff
- These routes are assigned the `web` middleware group, which provides features like session state and CSRF protection.
+ 这些路由分配 给`web` 中间件，而这个中间件为 `Session` 和 `CSRF` 提供了保护功能。
```

这些路由都被分配了 `web` 中间件，而这个中间件提供了像 `Session` 与 `CSRF` 保护之类的特性。

overtrue commented on routing.md:
```diff
@@ -1,39 +1,39 @@
+大多数的应用，都是从 `routes/web.php` 文件开始定义路由。
 
-#### Available Router Methods
+#### 正确的定义路由方法
```
`Available Router Methods` 应该是 `可用的路由方法` 吧？

summerblue replied:
建议采用

overtrue commented on routing.md:
```diff
@@ -52,39 +52,42 @@ Sometimes you may need to register a route that responds to multiple HTTP verbs.
+#### CSRF 保护
 
-Any HTML forms pointing to `POST`, `PUT`, or `DELETE` routes that are defined in the `web` routes file should include a CSRF token field. Otherwise, the request will be rejected. You can read more about CSRF protection in the [CSRF documentation](/docs/{{version}}/csrf):
+任何包含 `POST`, `PUT` 或 `DELETE` 使用 HTML 提交的路由，都在 `web` 路由中包含一个 CSRF 令牌，如果没有，这个请求将会被拒绝。更多的关于 CSRF 的说明在 [CSRF documentation](/docs/{{version}}/csrf):
```
``` diff
- 任何包含 `POST`, `PUT` 或 `DELETE` 使用 HTML 提交的路由，都在 `web` 路由中包含一个 CSRF 令牌，如果没有
+ 任何指向 `web` 中 `POST`, `PUT` 或 `DELETE` 路由的 HTML 表单请求都应该包含一个 CSRF 令牌，否则
```

overtrue commented on routing.md:
```diff
@@ -52,39 +52,42 @@ Sometimes you may need to register a route that responds to multiple HTTP verbs.
 
-Route parameters are always encased within `{}` braces and should consist of alphabetic characters. Route parameters may not contain a `-` character. Use an underscore (`_`) instead.
+
+路由的参数通常都会被放在「大括号」内。当运行路由时，参数会通过路由闭包来传递。 
```
这里的：

```
路由的参数通常都会被放在「大括号」内
```

不应该把 `{}` 省掉了，是不是这样会好些呢？

```
路由的参数通常都会被放在大括号 `{}` 内，并且参数名只能为字母
```

kzh4435 replied:
好的

overtrue commented on routing.md:
```diff
@@ -52,39 +52,42 @@ Sometimes you may need to register a route that responds to multiple HTTP verbs.
+### 可选路由参数
 
-Occasionally you may need to specify a route parameter, but make the presence of that route parameter optional. You may do so by placing a `?` mark after the parameter name. Make sure to give the route's corresponding variable a default value:
+当需要指定一个路由参数时，可以在参数后面加上 `?` 来实现，但是相应的变量必须有默认值：
```
``` diff
- 当需要指定一个路由参数时，可以在参数后面加上 `?` 来实现
+ 当需要指定一个路由参数为可选时，可以在参数后面加上 `?` 来实现
```

overtrue commented on routing.md:
```diff
@@ -95,29 +98,29 @@ Occasionally you may need to specify a route parameter, but make the presence of
+#### 为命名路由生成 URLs
 
-Once you have assigned a name to a given route, you may use the route's name when generating URLs or redirects via the global `route` function:
+为路由指定了名称后，在生成 URLs 或者重定向时，可以使用全局方法 `route` 来实现：  
```
``` diff
- 为路由指定了名称后，在生成 URLs 或者重定向时，可以使用全局方法 `route` 来实现：
+ 为路由指定了名称后，我们可以使用全局方法 `route` 来在生成 URLs 或者重定向到该条路由：
```

overtrue commented on routing.md:
```diff
@@ -95,29 +98,29 @@ Occasionally you may need to specify a route parameter, but make the presence of
     return redirect()->route('profile');
 
-If the named route defines parameters, you may pass the parameters as the second argument to the `route` function. The given parameters will automatically be inserted into the URL in their correct positions:
+如果是有定义参数的命名路由，可以把参数放到 `route` 方法的第二个参数位置，参数会自动插入到 URL 中：
```
``` diff
- 如果是有定义参数的命名路由，可以把参数放到 `route` 方法的第二个参数位置，参数会自动插入到 URL 中
+ 如果是有定义参数的命名路由，你可以把参数作为 `route` 方法的第二个参数传入，指定的参数将会自动插入到 URL 中对应的位置
```

nauxliu commented on routing.md:
```diff
@@ -42,7 +42,7 @@ The router allows you to register routes that respond to any HTTP verb:
     Route::options($uri, $callback);
 
-Sometimes you may need to register a route that responds to multiple HTTP verbs. You may do so using the `match` method. Or, you may even register a route that responds to all HTTP verbs using the `any` method:
+有的时候你可能需要注册一个可响应多个 HTTP 动作的路由，这时可以使用 `match` 方法，也可以使用 `any` 方法注册一个实现响应所有 HTTP 的请求的路由：
```
``` diff
- The router allows you to register routes that respond to any HTTP verb:
+ 我们可以注册路由来响应所有的 HTTP 请求

- Sometimes you may need to register a route that responds to multiple HTTP verbs.
+ 有的时候你可能需要注册一个可响应多个 HTTP 动作的路由
```

两处的 `HTTP verb` 可以统一下翻译，个人觉得此处的 `verb` 统一翻译为 `方法` 比较合适。

overtrue commented on routing.md:
```diff
@@ -126,14 +129,15 @@ If the named route defines parameters, you may pass the parameters as the second
 
 <a name="route-groups"></a>
-## Route Groups
+## 路由群组
```
路由组

overtrue commented on routing.md:
```diff
@@ -126,14 +129,15 @@ If the named route defines parameters, you may pass the parameters as the second
 
-Route groups allow you to share route attributes, such as middleware or namespaces, across a large number of routes without needing to define those attributes on each individual route. Shared attributes are specified in an array format as the first parameter to the `Route::group` method.
+
+路由群组允许共享路由属性，例如中间件和命名空间等，我们没有必要为每个路由单独设置共同属性，共享属性会以数组的形式放到 `Route::group` 方法的第一个参数中。
```
这里的 `共同属性` 与 `共享属性` 我觉得可以统一一下，使用 `共有属性` 会不会好些呢？

overtrue commented on routing.md:
```diff
@@ -146,20 +150,21 @@ To assign middleware to all routes within a group, you may use the `middleware`
     });
 
-Remember, by default, the `RouteServiceProvider` includes your route files within a namespace group, allowing you to register controller routes without specifying the full `App\Http\Controllers` namespace prefix. So, you only need to specify the portion of the namespace that comes after the base `App\Http\Controllers` namespace.
+请记住，默认 `RouteServiceProvider` 会在命名空间群组中引入你的路由文件，让你不用指定完整的 `App\Http\Controllers` 命名空间前缀就能注册控制器路由，因此，我们在定义的时候只需要指定命名空 `App\Http\Controllers` 以后的部分。
```
``` diff
- 因此，我们在定义的时候只需要指定命名空
+ 因此，我们在定义的时候只需要指定命名空间
```

overtrue commented on routing.md:
```diff
@@ -146,20 +150,21 @@ To assign middleware to all routes within a group, you may use the `middleware`
 
-Route groups may also be used to handle sub-domain routing. Sub-domains may be assigned route parameters just like route URIs, allowing you to capture a portion of the sub-domain for usage in your route or controller. The sub-domain may be specified using the `domain` key on the group attribute array:
+路由群组也可以用作子域名的通配符，子域名可以像 URIs 一样当作路由群组的参数，因此允许把捕获的子域名一部分用于我们的路由或控制器。
+路由群组中子域名的属性可以使用群组属性的 `domain` 键。
```
原文这里没有换行哦：

``` diff
- 路由群组也可以用作子域名的通配符，子域名可以像 URIs 一样当作路由群组的参数，因此允许把捕获的子域名一部分用于我们的路由或控制器。
- 路由群组中子域名的属性可以使用群组属性的 `domain` 键。
+ 路由群组也可以用作子域名的通配符，子域名可以像 URIs 一样当作路由群组的参数，因此允许把捕获的子域名一部分用于我们的路由或控制器。路由群组中子域名的属性可以使用群组属性的 `domain` 键。
```

overtrue commented on routing.md:
```diff
@@ -179,24 +184,26 @@ The `prefix` group attribute may be used to prefix each route in the group with
+## 路由模型绑定
 
-When injecting a model ID to a route or controller action, you will often query to retrieve the model that corresponds to that ID. Laravel route model binding provides a convenient way to automatically inject the model instances directly into your routes. For example, instead of injecting a user's ID, you can inject the entire `User` model instance that matches the given ID.
+当注入模型 ID 到路由控制器时，我们通常需要查询这个 ID 的模型，Laravel 路由模型绑定提供了一个方便的方法自动将模型注入到我们的路由中，例如，除了注入一个用户的 ID，你也可以注入与指定 ID 相符的完整 `User` 类实例。
```
``` diff
- 我们通常需要查询这个 ID 的模型
+ 我们通常需要查询这个 ID 对应的模型
```

overtrue commented on routing.md:
```diff
@@ -179,24 +184,26 @@ The `prefix` group attribute may be used to prefix each route in the group with
-In this example, since the Eloquent `$user` variable defined on the route matches the `{user}` segment in the route's URI, Laravel will automatically inject the model instance that has an ID matching the corresponding value from the request URI. If a matching model instance is not found in the database, a 404 HTTP response will automatically be generated.
+ 在这个例子中，由于类型声明了 Eloquent 模型 `App\User`，对应的变量名 `$user` 会匹配路由片段中的 `{user}`，这样，Laravel 会自动注入与请求 URI 中传入的 ID 对应的用户模型实例。
+
+如果数据库中找不到对应的模型实例，会会自动生成 HTTP 404 响应。
```
``` diff
- 如果数据库中找不到对应的模型实例，会会自动生成 HTTP 404 响应。
+ 如果数据库中找不到对应的模型实例，将会自动生成产生一个 404 HTTP 响应。
```

overtrue commented on routing.md:
```diff
@@ -209,58 +216,59 @@ If you would like implicit model binding to use a database column other than `id
+### 显式绑定
 
-To register an explicit binding, use the router's `model` method to specify the class for a given parameter. You should define your explicit model bindings in the `boot` method of the `RouteServiceProvider` class:
+要注册显式绑定，需要使用路由的 model 方法来为给定参数指定绑定类。应该在 RouteServiceProvider::boot 方法中定义模型绑定：
```
``` diff
- 要注册显式绑定，需要使用路由的 model 方法来为给定参数指定绑定类。应该在 RouteServiceProvider::boot 方法中定义模型绑定：
+ 要注册显式绑定，你需要在 RouteServiceProvider::boot 方法中使用路由的 model 方法来为给定参数指定绑定类：
```

overtrue commented on routing.md:
```diff
@@ -209,58 +216,59 @@ If you would like implicit model binding to use a database column other than `id
+因为我们已经绑定 `{user}` 参数至 `App\User` 模型，所以 `User` 实例会被注入至该路由。所以，举个例子，一个至 `profile/1` 的请求会注入 ID 为 1 的 `User` 实例。
 
-If a matching model instance is not found in the database, a 404 HTTP response will be automatically generated.
+> **注意：**如果符合的模型不存在于数据库中，就会自动抛出一个 404 异常。
```
``` diff
- 如果符合的模型不存在于数据库中
+ 如果在数据库不存在对应 ID 的数据
```

overtrue commented on routing.md:
```diff
@@ -209,58 +216,59 @@ If you would like implicit model binding to use a database column other than `id
+## 表单方法伪造
 
-HTML forms do not support `PUT`, `PATCH` or `DELETE` actions. So, when defining `PUT`, `PATCH` or `DELETE` routes that are called from an HTML form, you will need to add a hidden `_method` field to the form. The value sent with the `_method` field will be used as the HTTP request method:
+HTML 表单没有支持 `PUT`、`PATCH` 或 `DELETE` 动作。所以在从 HTML 表单中调用被定义的 `PUT`、`PATCH` 或 `DELETE` 路由时，你将需要在表单中增加隐藏的 `_method` 字段。跟随 `_method` 字段送出的值将被作为 HTTP 的请求方法使用：
```
``` diff
- 跟随 `_method` 字段送出的值
+ `_method` 的值
```

overtrue commented on routing.md:
```diff
@@ -209,58 +216,59 @@ If you would like implicit model binding to use a database column other than `id
     </form>
 
-You may use the `method_field` helper to generate the `_method` input:
+你也可以使用 `methid_field` 辅助函数来生成隐藏的输入字段 `_method`：
```
``` diff
- 你也可以使用 `methid_field` 辅助函数
+ 你也可以使用辅助函数 `methid_field` 
```

overtrue commented on routing.md:
```diff
@@ -209,58 +216,59 @@ If you would like implicit model binding to use a database column other than `id
 
 <a name="accessing-the-current-route"></a>
-## Accessing The Current Route
+##  获取当前路由信息
```
``` diff
-  获取当前路由信息
+ 获取当前路由信息
```

overtrue commented on routing.md:
```diff
@@ -209,58 +216,59 @@ If you would like implicit model binding to use a database column other than `id
     $action = Route::currentRouteAction();
 
-Refer to the API documentation for both the [underlying class of the Route facade](http://laravel.com/api/{{version}}/Illuminate/Routing/Router.html) and [Route instance](http://laravel.com/api/{{version}}/Illuminate/Routing/Route.html) to review all accessible methods.
+更多 API 请参考 [underlying class of the Route facade](http://laravel.com/api/{{version}}/Illuminate/Routing/Router.html) and [Route instance](http://laravel.com/api/{{version}}/Illuminate/Routing/Route.html) 两个文件
```
``` diff
- 两个文件
```

kzh4435 replied:
幸苦了。是修改以后重新提交吗？

overtrue replied:
不，直接在你现有的代码上修改，修改完 commit, push 就好了，这里会同时更新。

kzh4435 committed:
Update routing.md

overtrue commented on routing.md:
```diff
@@ -1,39 +1,39 @@
+- [路由模型绑定](#route-model-binding)
+    - [隐式绑定](#implicit-binding)
+    - [显式绑定](#explicit-binding)
+- [跨站请求伪造](#form-method-spoofing)
```
忘记改了

overtrue commented on routing.md:
```diff
@@ -1,39 +1,39 @@
+#### 默认路由文件
 
-All Laravel routes are defined in your route files, which are located in the `routes` directory. These files are automatically loaded by the framework. The `routes/web.php` file defines routes that are for your web interface. These routes are assigned the `web` middleware group, which provides features like session state and CSRF protection. The routes in `routes/api.php` are stateless and are as ...
+所有的 Laravel 路由定义都在 `routes` 目录。路由文件是框架自动加载的。 `routes/web.php` 文件定义了所有的 web 界面的路由。这些路由分配 给`web` 中间件，而这个中间件提供了像 `Session` 和 `CSRF` 保护之类的特性。 `routes/api.php` 中的路由是没有状态的，它分配给了 `api` 中间件。
```
``` diff
- 分配 给`web` 中间件
+ 分配 给 `web` 中间件
```

overtrue commented on routing.md:
```diff
@@ -95,29 +98,29 @@ Occasionally you may need to specify a route parameter, but make the presence of
+#### 为命名路由生成 URLs
 
-Once you have assigned a name to a given route, you may use the route's name when generating URLs or redirects via the global `route` function:
+为路由指定了名称后，我们可以使用全局方法 `route` 来在生成 URLs 或者重定向到该条路由：  
```
``` diff
- 全局方法
+ 全局辅助函数
```

overtrue commented on routing.md:
```diff
@@ -95,29 +98,29 @@ Occasionally you may need to specify a route parameter, but make the presence of
     return redirect()->route('profile');
 
-If the named route defines parameters, you may pass the parameters as the second argument to the `route` function. The given parameters will automatically be inserted into the URL in their correct positions:
+如果是有定义参数的命名路由，可以把参数作为 `route` 方法的第二个参数传入，指定的参数将会自动插入到 URL 中对应的位置：
```
类中才叫方法，这应该叫函数：

``` diff
- 方法
+ 函数
```

overtrue commented on routing.md:
```diff
@@ -146,20 +150,20 @@ To assign middleware to all routes within a group, you may use the `middleware`
+### 命名空间
 
-Another common use-case for route groups is assigning the same PHP namespace to a group of controllers using the `namespace` parameter in the group array:
+另一个常见的例子是，指定相同的 PHP 命名空间给控制器组。可以使用 namespace 参数来指定组内所有控制器的公共命名空间：
```
``` diff
- namespace
+ `namespace`
```

overtrue commented on routing.md:
```diff
@@ -168,9 +172,9 @@ Route groups may also be used to handle sub-domain routing. Sub-domains may be a
+### 路由前缀
 
-The `prefix` group attribute may be used to prefix each route in the group with a given URI. For example, you may want to prefix all route URIs within the group with `admin`:
+通过路由群组数组属性中的 `prefix` 键可以给每个路由群组的中路由加上指定的  URI 前缀，例如，我们可以给路由群组中所有的 URIs 加上路由前缀 `admin` :
```
``` diff
- `admin` :
+ `admin`:
```

kzh4435 committed:
修改了form-method-spoofing

第一版问题较多，超哥提出的意见很完善，先mark

overtrue commented on routing.md:
```diff
@@ -209,58 +215,59 @@ If you would like implicit model binding to use a database column other than `id
     $action = Route::currentRouteAction();
 
-Refer to the API documentation for both the [underlying class of the Route facade](http://laravel.com/api/{{version}}/Illuminate/Routing/Router.html) and [Route instance](http://laravel.com/api/{{version}}/Illuminate/Routing/Route.html) to review all accessible methods.
+更多 API 请参考 [underlying class of the Route facade](http://laravel.com/api/{{version}}/Illuminate/Routing/Router.html) and [Route instance](http://laravel.com/api/{{version}}/Illuminate/Routing/Route.html)
```
``` diff
- and
+ 和
```

kzh4435 committed:
edit 表单伪造

edit 表单伪造

kzh4435 committed:
模拟http请求修改为表单方法伪造

模拟http请求修改为表单方法伪造

kzh4435 committed:
修改了默认路由文件语句

修改了默认路由文件语句

kzh4435 committed:
把方法修改成函数

把方法修改成函数

JobsLong commented on routing.md:
```diff
@@ -52,39 +52,42 @@ Sometimes you may need to register a route that responds to multiple HTTP verbs.
+### 必选路由参数
 
-Of course, sometimes you will need to capture segments of the URI within your route. For example, you may need to capture a user's ID from the URL. You may do so by defining route parameters:
+当然，有时我们需要在路由中捕获一些 URL 片段。例如，我们需要从 URL 中捕获用户的 ID ，我们可以这样定义路由参数：
```
segments of the `URI`, not `URL`

JobsLong commented on routing.md:
```diff
@@ -95,29 +98,29 @@ Occasionally you may need to specify a route parameter, but make the presence of
+## 命名路由
 
-Named routes allow the convenient generation of URLs or redirects for specific routes. You may specify a name for a route by chaining the `name` method onto the route definition:
+命名路由可以方便的生成 URLs 或者重定向，可以在定义路由后使用 `name` 方法实现：
```
这里要不要把 `使用` 这个词换成 `链式调用` 呢？

kzh4435 replied:
链式调用有什么特殊意思吗，使用比较容易理解吧。

JobsLong commented on routing.md:
```diff
@@ -209,58 +214,59 @@ If you would like implicit model binding to use a database column other than `id
     }
```
这里示例代码不完整：

```
/**
 * Get the route key for the model.
 *
 * @return string
 */
public function getRouteKeyName()
{
    return 'slug';
}

```

kzh4435 replied:
请明确一下，我对比了下，发现一样呀

JobsLong commented on routing.md:
```diff
@@ -209,58 +214,59 @@ If you would like implicit model binding to use a database column other than `id
+## 表单方法伪造
 
-HTML forms do not support `PUT`, `PATCH` or `DELETE` actions. So, when defining `PUT`, `PATCH` or `DELETE` routes that are called from an HTML form, you will need to add a hidden `_method` field to the form. The value sent with the `_method` field will be used as the HTTP request method:
+HTML 表单没有支持 `PUT`、`PATCH` 或 `DELETE` 动作。所以在从 HTML 表单中调用被定义的 `PUT`、`PATCH` 或 `DELETE` 路由时，你将需要在表单中增加隐藏的 `_method` 字段。 `_method` 字段送出的值将被作为 HTTP 的请求方法使用：
```
`_method` 字段送出的值 感觉翻译的不是很好，还不如直接翻译成：

`_method` 字段的值

kzh4435 replied:
嗯，就这样改。

ghost commented:
已完成 `Review`

kzh4435 committed:
passwords 翻译第一版

passwords 翻译第一版

kzh4435 committed:
修改了部分语法错误

修改了部分语法错误

kzh4435 committed:
修改了重置密码说明的提示

修改了重置密码说明的提示

kzh4435 committed:
修改了重置密码说明

或者任何一个你使用 Laravel 应用-->或者任意一个在应用中分配的 URL

kzh4435 committed:
修改了start翻译

start-->CODING

summerblue commented on routing.md:
```diff
@@ -209,58 +214,59 @@ If you would like implicit model binding to use a database column other than `id
     $action = Route::currentRouteAction();
 
-Refer to the API documentation for both the [underlying class of the Route facade](http://laravel.com/api/{{version}}/Illuminate/Routing/Router.html) and [Route instance](http://laravel.com/api/{{version}}/Illuminate/Routing/Route.html) to review all accessible methods.
+更多 API 请参考 [underlying class of the Route facade](http://laravel.com/api/{{version}}/Illuminate/Routing/Router.html) 和 [Route instance](http://laravel.com/api/{{version}}/Illuminate/Routing/Route.html)
```
> 更多 API 请参考 [underlying class of the Route facade](http://laravel.com/api/{{version}}/Illuminate/Routing/Router.html) 和 [Route instance](http://laravel.com/api/{{version}}/Illuminate/Routing/Route.html)

修改为：

> 完整的方法列表请参考 [Route facade](http://laravel.com/api/{{version}}/Illuminate/Routing/Router.html) 和 [Route 实例](http://laravel.com/api/{{version}}/Illuminate/Routing/Route.html) 的 API 文档。

kzh4435 replied:
好的，这样更友好

summerblue commented on routing.md:
```diff
@@ -209,58 +214,59 @@ If you would like implicit model binding to use a database column other than `id
+### 显式绑定
 
-To register an explicit binding, use the router's `model` method to specify the class for a given parameter. You should define your explicit model bindings in the `boot` method of the `RouteServiceProvider` class:
+要注册显式绑定，你需要在 RouteServiceProvider::boot 方法中使用路由的 model 方法来为给定参数指定绑定类：
```
> explicit model bindings in the `boot` method of the `RouteServiceProvider` class

这里请注意代码快必须使用代码高亮：

```
`RouteServiceProvider::boot` 
```

此句话翻译可用使用英文翻译常用的「本末倒置大法」：

```
To register an explicit binding, use the router's `model` method to specify the class for a given parameter.
```

翻译为：

```
可以使用 `router` 的 `model` 方法来注册显式绑定。
```

这句话，建议这么翻译：

```
To register an explicit binding, use the router's `model` method to specify the class for a given parameter. You should define your explicit model bindings in the `boot` method of the `RouteServiceProvider` class:
```

译为

```
可以使用 `router` 的 `model` 方法来注册显式绑定。你应该在 `RouteServiceProvider` 类中的 `boot` 方法内定义这些显式绑定：
```

kzh4435 replied:
好的，有时自己也觉得绕口，但是不知道如何措辞。

summerblue commented on passwords.md:
```diff
@@ -1,61 +1,63 @@
+## 重置密码说明
 
-> {tip} **Want to get started fast?** Just run `php artisan make:auth` in a fresh Laravel application and navigate your browser to `http://your-app.dev/register` or any other URL that is assigned to your application. This single command will take care of scaffolding your entire authentication system, including resetting passwords!
+> ｛提示｝ **想要快速的 CODING ？** 首先在 Laravel 应用下运行 `php artisan make:auth` 命令，然后使用浏览器打开 `http://your-app.dev/register` ，或者任意一个在应用中分配的 URL 。这个命令将会生成包括密码重置在内的整个认证系统。
```
不建议使用 `CODING` 哈，我们本来就是在做翻译。

```
**Want to get started fast?** 
```

建议翻译为

```
**想要快速上手此功能？**
```

kzh4435 replied:
好的，这就改下，他们这个start在每个地方都是不同意思。

summerblue commented on passwords.md:
```diff
@@ -78,9 +82,9 @@ In your `auth.php` configuration file, you may configure multiple password "brok
+#### 自定义重置邮箱
 
-You may easily modify the notification class used to send the password reset link to the user. To get started, override the `sendPasswordResetNotification` method on your `User` model. Within this method, you may send the notification using any notification class you choose. The password reset `$token` is the first argument received by the method:
+你可以方便的修改通知类用以给用户发送密码重置链接。动手 CODING ，在 `User` 模型中重写  `sendPasswordResetNotification` 方法。在这个方法中，你可以使用你选择的任意方法发送通知。密码重置 `$token` 是这个方法的接收第一个参数：
```
```
To get started,
```

还是 `CODING` 😃 ，建议修改为：

```
在开始之前，
```

kzh4435 committed:
修改了routing 的250，272

修改了routing 的最下面的帮助

kzh4435 committed:
修改 218  219 

修改了218 代码加高亮，219重新翻译

kzh4435 committed:
修改了 start翻译

修改了 start翻译
1.不建议使用 CODING 哈，我们本来就是在做翻译。

**Want to get started fast?** 
建议翻译为

**想要快速上手此功能？**

2.To get started,
还是 CODING :smiley: ，建议修改为：

在开始之前。

kzh4435 committed:
18 use加高亮

18 use加高亮

kzh4435 committed:
添加翻译署名

添加翻译署名

kzh4435 committed:
修改图片大小

修改图片大小
